NativeApp/src/components/Home/container.js

```diff
@@ -43,14 +39,12 @@ export default Container => class extends Component {
 
     onDayPress = (day) => {
       const { navigation } = this.props;
+      const { takenHolidays } = this.state;
+
       if (day) {
-        this.setState({
-          booking: {
-            startDate: day.dateString,
-            endDate: day.dateString,
-          },
-        });
-        navigation.push('Booking', { date: day.dateString });
+        const booked = has(takenHolidays, day.dateString);
+        const holiday = get(takenHolidays, day.dateString, 0);
```

Comment:
Is there any way you can use the value returned from `const holiday = get(takenHolidays, day.dateString, 0);` to find out `booked`, rather than making two searches through the takenHolidays object?

Reply:
Not 100% sure, for the dates are set as keys within the `takenHoliday` object. The holiday returns an object containing details from the matching selected date, so for example `{textColor: "white", color: "#ff9b34", holId: 1}`.

Booked looks to see if the `takenHolidays` object contains the selected date. It returns a boolean.

I need both values passed down, this was the best way I could think of doing it.

Reply:
Example of `takenHolidays` object.
`2018-07-01: {textColor: "white", color: "#ff9b34", holId: 1}`

Reply:
I see. I thought something like this might be the case which is why I still approved 👍